src/rpc/blockchain.cpp

```diff
@@ -1693,6 +1695,24 @@ static UniValue getblockstats(const JSONRPCRequest& request)
             "  \"txs\": xxxxx,             (numeric) The number of transactions (excluding coinbase)\n"
             "  \"utxo_increase\": xxxxx,   (numeric) The increase/decrease in the number of unspent outputs\n"
             "  \"utxo_size_inc\": xxxxx,   (numeric) The increase/decrease in size for the utxo index (not discounting op_return and similar)\n"
+            "  \"nested_p2wpkh_outputs_spent\": xxxxx,   (numeric) The total number of nested P2WPKH outputs spent\n"
+            "  \"nested_p2wsh_outputs_spent\": xxxxx,   (numeric) The total number of nested P2WSH outputs spent\n"
+            "  \"native_p2wpkh_outputs_spent\": xxxxx,   (numeric) The total number of native P2WPKH outputs spent\n"
+            "  \"native_p2wsh_outputs_spent\": xxxxx,   (numeric) The total number of native P2WSH outputs spent\n"
+            "  \"txs_spending_nested_p2wpkh_outputs\": xxxxx,   (numeric) The total number of transactions spending nested P2WPKH outputs\n"
+            "  \"txs_spending_nested_p2wsh_outputs\": xxxxx,   (numeric) The total number of transactions spending nested P2WSH outputs\n"
+            "  \"txs_spending_native_p2wpkh_outputs\": xxxxx,   (numeric) The total number of transactions spending native P2WPKH outputs\n"
+            "  \"txs_spending_native_p2wsh_outputs\": xxxxx,   (numeric) The total number of transactions spending native P2WSH outputs\n"
+            "  \"new_p2wpkh_outputs\": xxxxx,   (numeric) The total number of new P2WPKH outputs\n"
+            "  \"new_p2wsh_outputs\": xxxxx,   (numeric) The total number of new P2WSH outputs\n"
+            "  \"txs_creating_p2wpkh_outputs\": xxxxx,   (numeric) The total number of transactions creating new P2WPKH outputs\n"
+            "  \"txs_creating_p2wsh_outputs\": xxxxx,   (numeric) The total number of transactions new P2WSH outputs\n"
+            "  \"txs_signalling_opt_in_rbf\": xxxxx,   (numeric) The total number of transactions that signal RBF\n"
+            "  \"txs_consolidating\": xxxxx,   (numeric) The total number of consolidating transactions (defined as at least 3 inputs and 1 output)\n"
```

Comment:
Might be worth returning a list of txids instead of a raw count, and maybe consider same for some of the other added keys. This'd bloat the size of the returned JSON by a bit, but gives much more information to the user.